script.js

```diff
@@ -11,3 +11,133 @@ closeBtn.onclick = function closeMenu() {
+    workTools: ['html', 'css', 'javaScript', 'github', 'Ruby', 'Bootstrap'],
+  },
+];
+
+const dynamic = document.querySelector('.popup');
+dynamic.innerHTML = `<div class="row">
+                <div class="right-col work-1">
+                  <a href="#"><h2>${portfolioData[0].title}</h2></a>
+                <div class="work-details flex2">
+                  <a href="#"><h3>${portfolioData[0].workDetails[0]}</h3></a>
+                  <div class="bullet-point"></div>
+                  <a href=""><h4>Back End Dev</h4></a>
+                  <div class="bullet-point"></div>
+                  <a href=""><h4>2015</h4></a>
+                  
+                </div>
+
+                <div class='close'>
+          <img src="./images/greyX.svg" alt="close" class='x-icon'
+            id="close"/>
+        </div>
+                  
+                <a href="#"
+                  ><img src="./images/Portfolio 1.png" class="mobile-only" alt="portfolio-1"
+                /></a>
+  
+                   <a href="#"
+                  ><img src="./images/pop-img1.png" class="desktop-only" alt="portfolio-1"
+                /></a>
+                
+                </div>
+
+                <div class="left-col">
+                <div class="text-side">
+
+                <p class="desktop-only" >
+                  Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea
+                </p>
+                <p class="mobile-only" >
+                  Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent
+                </p>
+                </div>
+                <div class="links-side">
+                <ul>
+                  <li class="work-tools">
+                    <button type="button" class="tools desktop-only">html</button>
```

Comment:
- [ ] Please note that all the data for the modal popup should be filled from the project details array, Modal popup data is populated with the data from the projects array. This will make adding projects to your portfolio quite easy

- [ ] Please also make sure that your work section is generated dynamically , The projects section is created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). **This requires changes in existing code - already created HTML code for projects list should be removed, and instead it should be added via JavaScript.**